src/viem/prepareTransactions.test.ts

```diff
@@ -177,7 +227,7 @@ describe('prepareTransactions module', () => {
       const result = await prepareTransactions({
         feeCurrencies: mockFeeCurrencies,
         spendToken: mockFeeCurrencies[1],
-        spendTokenAmount: new BigNumber(100_000),
+        spendTokenAmount: new BigNumber(9_000),
```

Comment:
This isn't more than the max amount for `mockFeeCurrencies[1]` no?
Maybe we should update the test description then? since we can't try to prepare more than the spend balance anymore.
Or even remove it?

Reply:
yeah it doesn't look like this test adds useful coverage anymore given the other tests that we have and the expected behavior now. I removed it